modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -25,29 +23,20 @@ The properties for broker-wide throughput quota balancing are configured at the
 | xref:reference:cluster-properties.adoc#kafka_throughput_limit_node_out_bps[kafka_throughput_limit_node_out_bps]
 | A broker's total throughput limit for egress Kafka traffic.
 
```

Comment:
Should document / reference `kafka_throughput_controlled_api_keys` and `kafka_throughput_control` (both cluster properties)

Reply:
Are these new properties? We don't have them in the cluster properties list. Can you help flesh out details for them?

Specifically, if appropriate:
Type
Units
Default
Range
Restart required

Reply:
Not new.

### `kafka_throughput_controlled_api_keys`
Source: https://github.com/redpanda-data/redpanda/blob/v23.3.13/src/v/config/configuration.cc#L2852-L2858

Type: `list<string>`
Units: N/A
Default: `["produce", "fetch"]`
Range: N/A
Restart required: No
Visibility: User

### `kafka_throughput_control`
Source: https://github.com/redpanda-data/redpanda/blob/v23.3.13/src/v/config/configuration.cc#L2859-L2873
Units: N/A
Default: `[]`
Range: N/A
Restart required: No
Visibility: User
Example: `[{'name': 'first_group','client_id': 'client1'}, {'client_id': 'consumer-\d+'}, {'name': 'catch all'}]`

Reply:
Based on Ben's feedback - I have one open question on the kafka_throughput_control property. The code seems ambiguous as to how it behaves if you define a name but no client_id for a throughput control group. The example in the code is:

[{'name': 'first_group','client_id': 'client1'}, {'client_id': 'consumer-\d+'}, {'name': 'catch all'}]

I need to document the behavior for when the list has an entry with a name but no client_id. As it stands, I can omit that example, but it seems to be an important capability/distinction for configuring throughput throttling.